internal/parse/parse.go

```diff
@@ -49,16 +51,76 @@ var (
 	errDuplicateDest = errors.New("duplicate mount destination")
 )
 
+func mountIsReadWrite(m specs.Mount) bool {
```

Comment:
Not a biggy, but this was down around line 245 in https://github.com/containers/buildah/pull/5965/commits/8e0bc8272b5fd221e4a9623f93e3abf84ea41933#diff-6dfa1c75be9b4be84230714c1b18f042887775562dbe64be2c59ea562b9827e3R245  It shouldn't matter functionally, it just makes future backports a little more persnickety going forward.

Reply:
Making future backports easier is definitely a goal, I'll look into moving it.  Thanks for pointing this out.

Reply:
This will be moved in my next push.